ui/app/components/ui/search-icon/search-icon.component.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export default function SearchIcon () {
+  return (
+    <svg height="20" width="20" xmlns="http://www.w3.org/2000/svg" fill="currentColor">
+      <g clipRule="evenodd" fillRule="evenodd">
```

Comment:
Oh I see, this uses `currentColor`. That's handy. I see why you used an `svg` tag instead of `img` now.

Maybe we'll consolidate the others to use this new `search-icon` instead! I like this idea.

Reply:
Yeah, it's pretty useful as then you can only use the single icon that depends on a font color. In general, would be even awesome to get rid of manually created components as that's a job for a bundler to handle appropriately (for e.g. https://github.com/smooth-code/svgr)